src/test/mocks/@interlay/interbtc-api/parachain/issue.ts

```diff
@@ -4,15 +4,13 @@ import { BitcoinAmount } from '@interlay/monetary-js';
 import { DEFAULT_ISSUE_DUST_AMOUNT } from '@/config/parachain';
 import { WRAPPED_TOKEN } from '@/config/relay-chains';
 
-const MOCK_ISSUE_REQUEST_LIMITS = {
-  singleVaultMaxIssuable: newMonetaryAmount('56527153', WRAPPED_TOKEN),
-  totalMaxIssuable: newMonetaryAmount('493817337', WRAPPED_TOKEN)
-};
-
 const mockIssueGetDustValue = jest.fn(() => new BitcoinAmount(DEFAULT_ISSUE_DUST_AMOUNT));
 
-const mockIssueGetRequestLimits = jest.fn(() => MOCK_ISSUE_REQUEST_LIMITS);
+const mockIssueGetRequestLimits = jest.fn(() => ({
+  singleVaultMaxIssuable: newMonetaryAmount('56527153', WRAPPED_TOKEN),
+  totalMaxIssuable: newMonetaryAmount('493817337', WRAPPED_TOKEN)
```

Comment:
I would put these number values into constant in case we wanted to use them in the tests later. Also is there a reason not to use more rounded number?

Reply:
Also a tip: if you use `new MonetaryAmount()` constructor you can specify amount in base denomination - so no need to put long atomic values - this might help to increase readability of values :)

Reply:
Thank you for your feedback. Let me update them accordingly.